src/test/scala/uk/gov/nationalarchives/dp/client/EntityTest.scala

```diff
@@ -21,15 +21,9 @@ abstract class EntityTest[F[_]](implicit cme: MonadError[F, Throwable])
   )
   forAll(entityTypes) { (entityType, expectedPath) =>
     "fromType" should s"return an object with path $expectedPath for entity type $entityType" in {
-      valueFromF(fromType(entityType, UUID.randomUUID(), None, deleted = false)).path should equal(
+      valueFromF(fromType(entityType, UUID.randomUUID(), None, deleted = false)).path.get should equal(
```

Comment:
Should we add the case for when there is an unexpected entityType? 
```
  val entityTypes: TableFor2[String, String] = Table(
    ("entityType", "expectedPath"),
    ("IO", "information-objects"),
    ("CO", "content-objects"),
    ("SO", "structural-objects")
    ("UnexpectedEntityType", "")
  )
```

Instead of `path.get`, you could do `path.getOrElse("")` or `path.getOrElse("no path")`?

Reply:
Good idea, I've added the other test case in. For that to work, I've changed the table to `TableFor2[String, Option[String]]` so I'm just comparing `path` with `expectedPath`